server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -112,6 +116,7 @@ public NodeStats(StreamInput in) throws IOException {
         ingestStats = in.readOptionalWriteable(IngestStats::read);
         adaptiveSelectionStats = in.readOptionalWriteable(AdaptiveSelectionStats::new);
         indexingPressureStats = in.readOptionalWriteable(IndexingPressureStats::new);
+        repositoriesThrottlingStats = in.readOptionalWriteable(RepositoriesThrottlingStats::new);
```

Comment:
we should handle the case where the node is communicating with a node in a different version. See the javadocs in `org.elasticsearch.TransportVersion` for more information.

Reply:
OK